migration-roadmap/slider.md

```diff
@@ -0,0 +1,525 @@
+    <div class="spectrum-Slider-content">
+        <!-- Slider controls -->
+        <div class="spectrum-Slider-controls">
+            <div class="spectrum-Slider-track" style="width: 40%;"></div>
+            <div class="spectrum-Slider-handle" style="left: 40%;">
+                <input
+                    type="range"
+                    id="slider-input-1"
+                    class="spectrum-Slider-input"
+                    value="14"
+                    step="2"
+                    min="10"
+                    max="20"
+                    tabindex="0"
+                />
+            </div>
+            <div class="spectrum-Slider-track" style="width: 60%;"></div>
+        </div>
+
+        <!-- Conditionally rendered textfield if slider is editable -->
+        <div class="spectrum-Textfield spectrum-Slider-editable">
+            <input type="text" class=" spectrum-Textfield-input " />
+        </div>
+    </div>
+</div>
+```
+
+</details>
+
+<details>
+<summary>Diff: Legacy (CSS main) → Spectrum 2 (CSS spectrum-two)</summary>
+
+```diff
+<div class="spectrum-Slider spectrum-Slider--sizeM" aria-labelledby="slider-label">
+  <!-- Label region -->
+  <div class="spectrum-Slider-labelContainer">
+    <label class=" spectrum-FieldLabel spectrum-Slider-label " id="slider-label" for="slider-1">
+      Slider label
+    </label>
+    <div role="textbox" aria-readonly="true" class="spectrum-Slider-value" aria-labelledby="slider-label">
+      14
+    </div>
+  </div>
+
++ <div class="spectrum-Slider-content">
+    <!-- Slider controls -->
+    <div class="spectrum-Slider-controls">
+      <div class="spectrum-Slider-track" style="width:40%;"></div>
+      <div class=" spectrum-Slider-handle " style="left:40%;">
+-       <input type="range" id="slider-input-1" class="spectrum-Slider-input" value="14" step="2" min="10" max="20">
++       <input type="range" id="slider-input-1" class="spectrum-Slider-input" value="14" step="2" min="10" max="20" tabindex="0"/>
+      </div>
+      <div class="spectrum-Slider-track" style="width:60%;"></div>
+    </div>
++
++   <!-- Conditionally rendered textfield if slider is editable -->
++   <div class="spectrum-Textfield spectrum-Slider-editable">
++     <input type="text" class=" spectrum-Textfield-input ">
++   </div>
++ </div>
+</div>
+```
+
+**Key Changes in HTML Structure:**
+
+1. **Content wrapper introduction**: Added `.spectrum-Slider-content` container that wraps both the slider controls and the optional editable text field, providing better structural organization, as well as supporting the side-label layout.
+
+2. **Editable field integration**: Introduced conditional `.spectrum-Textfield.spectrum-Slider-editable` container with text input for editable slider functionality, enabling users to directly input values.
+
+3. **Enhanced accessibility**: Added explicit `tabindex="0"` to the slider handle for improved keyboard navigation and focus management.
+
+</details>
+
+### CSS => SWC mapping
```

Comment:
We're missing the precise (handle) variant, it's new for S2! On the table and in the summary.